test/lib/runner_test.rb

```diff
@@ -2,7 +2,7 @@
 
 class RunnerTest < Minitest::Test
   def setup
-    @runner = AuroraBootstrapParallelization::Runner.new( config: File.expand_path( "../../config.yml", __FILE__ ) )
+    @runner = AuroraBootstrapParallelization::Runner.new( {:config => File.expand_path( "../../config.yml", __FILE__ ), :cronjob => false} )
```

Comment:
The ruby 2+ preferred syntax is 
```ruby
@runner = AuroraBootstrapParallelization::Runner.new( config: File.expand_path( "../../config.yml", __FILE__ ), cronjob: false )
```

when passing in a has of params the `{}` are optional and `=>` are unnecessary for symbols, making it `symbol: value` instead of `:symbol => value`. but they're functionally identical